### PR TITLE
Fixing Jenkins rolebinding name to avoid conflicts

### PR DESCRIPTION
**Pull request by oybed**

### CODE REVIEW

File: openshift-templates/automation-api-hsql/template.json [hunk 1]
```diff
@@ -174,7 +174,7 @@
             "apiVersion": "v1",
             "kind": "RoleBinding",
             "metadata": {
-                "name": "edit"
+                "name": "jenkins_edit"
```
Comment (sherl0cks) [inline openshift-templates/automation-api-hsql/template.json]: I think this has the potential to class with https://github.com/openshift/origin/blob/master/examples/jenkins/jenkins-persistent-template.json#L207.

Perhaps we use `jenkins_pipeline_edit`?

File: openshift-templates/http-app-deploy/template.json [hunk 2]
```diff
@@ -168,7 +168,7 @@
             "apiVersion": "v1",
             "kind": "RoleBinding",
             "metadata": {
-                "name": "edit"
+                "name": "jenkins_edit"
```
Comment (sherl0cks) [inline openshift-templates/http-app-deploy/template.json]: same comment here

File: params/hoverfly/deploy [hunk 3]
```diff
@@ -1,3 +1,3 @@
-NAMESPACE=labs-ci-cd
+NAMESPACE=labs-ci-cdacetux
```
Comment (sherl0cks) [inline params/hoverfly/deploy]: think we might have picked up some local changes that we don't want to share

File: run.sh [hunk 4]
```diff
@@ -46,8 +46,8 @@ else
 fi 
 
 docker run --rm -i \
-    -v $(pwd):/tmp/src:z \
-    -v $HOME/.kube:/root/.kube:z \
+    -v $(pwd):/tmp/src \
+    -v $HOME/.kube:/root/.kube \
```
Comment (sherl0cks) [inline run.sh]: is the `:z` problematic? I can't get docker mounts to work correctly without it...

Comment (oybed) [reply run.sh]: @sherl0cks yeah, in my environment it is ... I'll add it back though. Need to troubleshoot more later. We see the same for CASL contributors - it works for some, but not others

**Comment by sherl0cks** [comment]
CI system has some issues, manually testing shows this to work correctly.

**Pull request merged**